helpers/deploymentUtils.ts

```diff
@@ -126,3 +133,13 @@ export const getUnregisteredRewardsDistributors = async (
     }),
   );
 };
+
+export async function getBlockNumberOrTimestampBasedDeploymentInfo(network: string): Promise<DeploymentInfo> {
+  if (network == "hardhat" && process.env.RUN_TIMESTAMP_BASED_INTEGRATION_TESTS === "true") {
+    return { isTimeBased: true, blocksPerYear: "31536000" }; // Adjust the default values as needed
+  } else if (network === "bscmainnet" || network === "bsctestnet" || network == "hardhat") {
+    return { isTimeBased: false, blocksPerYear: "10512000" };
+  } else {
+    return { isTimeBased: true, blocksPerYear: "31536000" };
```

Comment:
Could we add these values in the hardhat config file, under the `networks` section? This way, we could have in that file if each network is time-based or not and the blocks per year (if needed), and we could read that info from the `hre.network` object, that would be passed to this function

Reply:
Done https://github.com/VenusProtocol/isolated-pools/commit/6b98b095e5e6b3322b6d0c825ba273612e49948a